docs/setup/self-hosted/docker.mdx

```diff
@@ -86,44 +68,63 @@ Now you can access the MindsDB editor by going to `127.0.0.1:47334` in your brow
   <img src="/assets/mindsdb-local-editor.png" />
 </p>
 
-## Install dependencies
+<Warning>
+If you experience any issues related to MKL or your training process does not complete, please add the `MKL_SERVICE_FORCE_INTEL` environment variable, as below.
+
+```bash
+docker run -e MKL_SERVICE_FORCE_INTEL=1 -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+```
+</Warning>
+
+### Install dependencies
 
 MindsDB integrates with numerous data sources and AI frameworks. To use any of the integrations, you should enure that the required dependencies are installed in the Docker container.
 
-Here is how to install the dependencies in the Docker container:
+<Info>
```

Comment:
Should the 'Manage Integrations' page be pointed out as the main method for installing dependencies? And should the other methods (via the editor and the Add 'Datasource' page) be included as well?
@ZoranPandovski What do you think?

Reply:
Installing dependencies is part of the setup - therefore, I'd keep it here.

As of now, I mentioned about the 'Manage Integrations' page as the main method, as other methods blend in the flow. But if you think, we can mention it as well.

Btw. Do you think we should get rid of the method presented below that requires a container to be restarted? Or just keep it as an alternative.

Reply:
Yes, that makes sense. Since the other methods are part of the flow, let's not mention it here.

I think it might be best to keep it as an alternative for two reasons,
1. Some of our users might already be accustomed to this method.
2. Certain advanced users might actually prefer it (unlikely).